src/app/docs/dropdown/dropdown-docs.component.html

```diff
@@ -0,0 +1,330 @@
+          caption="use dropdowns as form inputs."
+          captionType="danger"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        thumbnailPath="~/assets/img/guidelines/dropdown/anatomy.png"
+      >
+        <sky-docs-anatomy-item>Dropdown button</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Dropdown menu</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Action</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Options"
+    >
+      <sky-docs-design-guideline
+        heading="Button types"
+      >
+      <table class="sky-docs-table">
+          <tr>
+            <td class="sky-docs-table-cell">Default</td>
+            <td class="sky-docs-table-cell">
+                <sky-docs-design-guideline-thumbnail
+                  imageSource="~/assets/img/guidelines/dropdown/option-type-default.png"
+                >
+                </sky-docs-design-guideline-thumbnail>
+              </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">Context menu</td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/dropdown/option-type-context.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">Icon-only</td>
+            <td class="sky-docs-table-cell">
+                <sky-docs-design-guideline-thumbnail
+                  imageSource="~/assets/img/guidelines/dropdown/option-type-icon.png"
+                >
+                </sky-docs-design-guideline-thumbnail>
+              </td>
+          </tr>
+        </table>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline
+        heading="Button styles"
+      >
+        <p>Dropdown buttons can use the default or primary styles inherited from SKY UX buttons. For details about when to use the primary style, see the <a routerLink="/design/guidelines/page-layouts" fragment="primary-action-guidelines">primary action guidelines</a>.</p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Behavior and states"
+    >
+      <sky-docs-design-guideline
+        heading="Behavoir"
+      >
+        <p>
+          The <a routerLink="/components/popover">popover</a> component defines the behavior of dropdown menus.
+        </p>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline
+        heading="States"
+      >
+        <p>
+          Dropdown buttons inherit the styling for states such as hover and disabled from <a routerLink="/components/button">SKY UX buttons</a>.
```

Comment:
This will need to be a hard URL (`href`) pointing to the page on the main docs site.

Reply:
For this one, I assume the button link will change once we get it over to phase 2 (I noticed the phase 2 popover link is different than the phase 1 links). Do we have a system for flagging these links that will need to updated as we move over to phase 2?

Reply:
Good question. I think ideally the links would change, but we have redirects in place on the primary docs site that will automatically redirect the user to the newer docs sites if they click on the old links.